README.md

```diff
@@ -38,6 +39,17 @@ To build minified javascript for production:
 
     npm run build
 
+## Deploy
+
+To deploy this, after merging any new PRs, follow these steps:
+
+1. Have an ipfs daemon running: `ipfs daemon`.
+2. Kill your `npm run dev` script if you happen to have it running.
+3. `npm install && npm prune`
+4. `npm run publish`. This should open the published page on the gateway.
+5. Pin the hash: `ipfs pin add <hash>`
```

Comment:
you need to pin the hash to the gateway, if your server might go away... I believe in IRC that is just:

`!pin <hash>`